client/v2/algod/algod.go

```diff
@@ -80,8 +80,8 @@ func (c *Client) Block(round uint64) *Block {
 	return &Block{c: c, round: round}
 }
 
-func (c *Client) GetProof(round uint64, txid string) *GetProof {
-	return &GetProof{c: c, round: round, txid: txid}
+func (c *Client) GetTransactionProof(round uint64, txid string) *GetTransactionProof {
```

Comment:
is there a way to easily invoke this function with the non-default hashtype param?

Reply:
`GetTransactionProof(round, txid).HashType("sha512_256").Do(args)`

Reply:
See https://github.com/algorand/go-algorand-sdk/pull/374/files/0842b7ff7bbfc39426b42411b65a8cc707404b33#diff-54d7a8bea6031095db3bfcc1351bf79d7289963f51ba21f268b66684b41bd142R36